src/applications/personalization/dashboard-2/tests/unit/healthcare.unit.spec.jsx

```diff
@@ -5,18 +5,48 @@ import { expect } from 'chai';
 import { HealthCare } from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
 
 describe('HealthCare', () => {
-  describe('Manage all your prescriptions', () => {
-    const defaultProps = {
-      appointments: [],
-      authenticatedWithSSOe: true,
-      canAccessRx: false,
-      fetchConfirmedFutureAppointments: () => {},
-    };
+  const defaultProps = {
+    appointments: [],
+    authenticatedWithSSOe: true,
+    canAccessRx: false,
+    canAccessMessaging: true,
+    fetchFolder: () => {},
+    fetchConfirmedFutureAppointments: () => {},
+  };
 
-    it('when enrolled in VA health care, should not render when they have never had any prescriptions managed by the VA', () => {
-      const wrapper = mount(<HealthCare {...defaultProps} />);
-      expect(wrapper.find('Prescriptions').exists()).to.be.false;
-      wrapper.unmount();
+  describe('Prescriptions', () => {
+    describe('when enrolled in VA Health Care', () => {
+      it('should not render when they have never had any prescriptions managed by the VA', () => {
+        const wrapper = mount(<HealthCare {...defaultProps} />);
+        expect(wrapper.find('Prescriptions').exists()).to.be.false;
+        wrapper.unmount();
+      });
+    });
+  });
+
+  describe('Messaging', () => {
+    describe('when eligible for messaging', () => {
+      it('should render the unread messages count', () => {
+        defaultProps.unreadMessagesCount = 3;
+        const wrapper = mount(<HealthCare {...defaultProps} />);
+        expect(wrapper.text()).to.contain('You have 3 new messages');
+        wrapper.unmount();
+      });
+
+      it('should render "View all new messages"', () => {
+        const wrapper = mount(<HealthCare {...defaultProps} />);
+        expect(wrapper.text()).to.contain('View your new messages');
+        wrapper.unmount();
+      });
+    });
+
+    describe('when not eligible for messaging', () => {
+      it('should not render', () => {
+        defaultProps.canAccessMessaging = false;
```

Comment:
editing `defaultProps` directly could lead to problems since these tests run in a random order. I'd suggest making a `defaultProps()` function that returns a fresh copy of the props object and then edit that before passing to the component.